leo/commands/build.rs

```diff
@@ -92,7 +92,6 @@ pub struct BuildOptions {
 
 /// Compile and build program command.
 #[derive(StructOpt, Debug)]
-#[structopt(setting = structopt::clap::AppSettings::ColoredHelp)]
```

Comment:
Does the modified version support colors in the cli?

Reply:
i covered this in the original post for this PR

> simple pr that switches leo over to `clap` instead of `structopt` since `structopt` is now deprecated. the only real change to the code is that `AppSettings::ColoredHelp` is now set by default

Reply:
great